src/pages/Catalogue/Catalogue.tsx

```diff
@@ -28,7 +29,13 @@ export const Catalogue: React.FC = () => {
   const fetchGames = async () => {
     try {
       setIsLoading(true);
-      await fetchAPI.post<Game[]>("/games", filters).then(setGameList);
+      await fetchAPI.post<Game[]>("/games", filters)
+      .then(
+        (games) => setGameList(games.map((game) => (
+          { ...game, 
+            is_wished: user?.game_collections?.find((collection) => collection.game_id === game.id)?.is_wished || false
+          })))
+      ).catch(() => toast.error("Erreur lors de la récupération des jeux"));
```

Comment:
```suggestion
      const games = await fetchAPI.post<Game[]>("/games", filters)
      setGameList(games.map(game => {
        const collection = user?.game_collections?.find(c => c.game_id === game.id);
        return {
          ...game,
          is_wished: collection?.is_wished || false,
          is_owned: collection?.is_owned || false,
          be_notified: collection?.be_notified || false,
        };
      }));
```
Je pense que ça serait plus pratique, cela nous évitera de réécrire la grande ligne et par même occasion autant placer les trois propriétés vu qu'ils sont déjà mis en place dans le Game.ts

Reply:
👍